testsuite/integration-tests-spring-web/pom.xml

```diff
@@ -16,7 +16,7 @@
     <name>RESTEasy Main testsuite: Spring Web integration tests</name>
 
     <properties>
-        <version.org.springframework>5.0.14.RELEASE</version.org.springframework>
+        <version.org.springframework>5.3.7</version.org.springframework>
```

Comment:
Can these versions properties be moved to resteasy-dependencies-bom?

Reply:
It actually has to live in both places because of https://github.com/resteasy/Resteasy/blob/da6aaedc5e64f8403536e105cc0e451a1fc60535/testsuite/integration-tests-spring-web/deployment/pom.xml#L164-L203. Properties are not inherited from imported POM's.